x/crosschain/client/integrationtests/inbound_voter_test.go

```diff
@@ -268,6 +268,11 @@ func (s *IntegrationTestSuite) TestCCTXInboundVoter() {
 				}
 				signedTx := BuildSignedInboundVote(s.T(), val, s.cfg.BondDenom, account, message)
 				out, err = clitestutil.ExecTestCLICmd(broadcaster.ClientCtx, authcli.GetBroadcastCommand(), []string{signedTx.Name(), "--broadcast-mode", "sync"})
+
+				outStr := out.String()
+				if strings.Contains(outStr, "failed") {
+					fmt.Println(outStr)
+				}
```

Comment:
do we need the print here?

Reply:
I was debugging stuff I forgot to remove